File: Bronze/Bronze_4/C_files/Code/Missa_Reversal_21665.c
#include <stdio.h>

int main()
{
    int n, m, sum = 0;
    char arr1[101][101], arr2[101][101], tmp;
    
    scanf("%d %d\n", &n, &m);
    
    for (int i = 0; i < n; i++) {
        scanf("%s", &arr1[i]);
    }

    for (int i = 0; i < n; i++) {
        scanf("%s", &arr2[i]);
    }

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            if ((arr1[i][j] == 'B' && arr2[i][j] == 'W') || (arr1[i][j] == 'W' && arr2[i][j] == 'B'))
                continue;
            else
                sum++;
        }
    }

    printf("%d", sum);

    return 0;
}

File: Bronze/Bronze_4/Java_files/Code/Missa_Reversal_21665.java
import java.util.Scanner;

public class Missa_Reversal_21665 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt(), m = sc.nextInt(), sum = 0;
        char arr1[][] = new char[n][m];
        char arr2[][] = new char[n][m];
        for (int i = 0; i < n; i++) {
            String s = sc.next();
            for (int j = 0; j < m; j++)
                arr1[i][j] = s.charAt(j);
        }
        for (int i = 0; i < n; i++) {
            String s = sc.next();
            for (int j = 0; j < m; j++)
                arr2[i][j] = s.charAt(j);
        }
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                if ((arr1[i][j] == 'B' && arr2[i][j] == 'W') || (arr1[i][j] == 'W' && arr2[i][j] == 'B'))
                    continue;
                else
                    sum++;
            }
        }
        System.out.println(sum);
    }
}
